apps/content_script/appbar.ts

```diff
@@ -42,10 +42,8 @@ function showBlockedByBannedWords(): void {
     rows: 10,
   });
 
-  const bannedWordsDiv = document.getElementById(
-    "banned-words-div",
-  ) as HTMLDivElement;
-  bannedWordsDiv.appendChild(textarea);
+  const bannedWordsDiv = document.getElementById("banned-words-div");
+  bannedWordsDiv?.appendChild(textarea);
```

Comment:
@vibh1103 

If there is no element with id=banned-words-div (due to a typo), this code will simply be ignored instead of giving an error. This delays finding bugs. Please throw an error if there is no bannedWordsDiv.